### PR TITLE
Sage Intacct SDK base commit

### PR DESCRIPTION
**Pull request by ashwin1111**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,75 @@
-# sageintacct-sdk-py
+# SageIntacctSDK
```
Comment (Shwetabhk) [inline README.md:1]: Rename to `Sage Intacct SDK`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,75 @@
...
 Python SDK to access Sage Intacct web services
+
+*Warning*: This SDK is under active development and some of the contracts will likely evolve in the next couple of weeks. It is not yet production-ready. If you want to collaborate on this, please mail siva@fyle.in
```
Comment (Shwetabhk) [inline README.md:4]: can remove this line

File: sageintacctsdk/apis/accounts.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        data = {
+            'readByQuery': {
+                'object': 'GLACCOUNT',
+                'fields': '*',
+                'query': "{0} = '{1}'".format(field, value),
+                'pagesize': '1000'
+            }
+        }
+
+        return self._format_post_request(data)['data']
```
Comment (Shwetabhk) [inline sageintacctsdk/apis/accounts.py:44]: Why is this function called `format_post_request` If we're using this for both GET and POST
rather rename it to `format_request`

File: sageintacctsdk/apis/accounts.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+        Returns:
+            Dict of state of request with RECORDNO.
+        """
+        data = {
+            'create': {
+                'GLACCOUNT': data
+            }
+        }
+        return self._format_post_request(data)
```
Comment (Shwetabhk) [inline sageintacctsdk/apis/accounts.py:23]: if we're sending the request in this function
then the function name does not suggest that. It just says `format`
instead it should say `format_and_send_request`

File: setup.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+"""
+import setuptools
+
+with open('README.md', 'r') as f:
+    long_description = f.read()
+
+setuptools.setup(
+    name='sageintacctsdk',
+    version='0.0.0',
+    author='Shwetabh Kumar',
```
Comment (Shwetabhk) [inline setup.py:12]: Please rename to your name and email.
You're the Author of this sdk :)

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+Project setup file
+"""
+import setuptools
+
+with open('README.md', 'r') as f:
+    long_description = f.read()
+
+setuptools.setup(
+    name='sageintacctsdk',
+    version='0.0.0',
```
Comment (Shwetabhk) [inline setup.py:11]: This will start from 0.1.0

File: sageintacctsdk/sageintacctsdk.py [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+        # Initializing variables
+        self.__sender_id = sender_id
+        self.__sender_password = sender_password
+        self.__user_id = user_id
+        self.__company_id = company_id
+        self.__user_password = user_password
+
+        self.__access_token = None
+
+        self.api_base = ApiBase()
```
Comment (Shwetabhk) [inline sageintacctsdk/sageintacctsdk.py:30]: is it necessary to expose `ApiBase`?
If yes why?

File: sageintacctsdk/apis/__init__.py [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+from .contacts import Contacts
+from .locations import Locations
+from .employees import Employees
+from .accounts import Accounts
+from .expense_types import ExpenseTypes
+from .attachments import Attachments
+from .expense_reports import ExpenseReports
+
+__all_ = [
+    'ApiBase',
```
Comment (Shwetabhk) [inline sageintacctsdk/apis/__init__.py:14]: is`ApiBase` required?

File: sageintacctsdk/sageintacctsdk.py (hunk 7 above)
Comment (ashwin1111) [reply sageintacctsdk/sageintacctsdk.py:29]: dicussed;

File: sageintacctsdk/apis/__init__.py (hunk 8 above)
Comment (ashwin1111) [reply sageintacctsdk/apis/__init__.py:14]: dicussed;

**Pull request merged**